plugins/header-classifier/src/main/java/io/buoyant/http/classifiers/HeaderClassifierInitializer.java

```diff
@@ -0,0 +1,15 @@
+package io.buoyant.http.classifiers;
+
+import io.buoyant.linkerd.ResponseClassifierInitializer;
+
+public class HeaderClassifierInitializer extends ResponseClassifierInitializer {
+    @Override
+    public String configId() {
+        return "io.buoyant.headerClassifier";
```

Comment:
depending on how tutorial-y you want to get with this change, you could add comments in this file explaining the functionality that Initializers provide. 
For example, `configId` is an important method: it defines the value for `kind` in the linkerd config